parsers/link-design-tokens/README.md

```diff
@@ -66,31 +84,36 @@ Array<{id: string, type: string, value: string, name: string} & Record<any, any>
 [
   {
     "id": "1",
-    "type": "measurement",
+    "type": "color",
```

Comment:
Why change the example?

Reply:
Because the example had to match with my explanations from the previous sections. Also, this parser is not very useful when used with text styles and font because their link is already done under the hood. I wanted to present a more probable use case.